dnf/util.py

```diff
@@ -643,30 +643,16 @@ def _is_file_pattern_present(specs):
     return False
 
 
-def is_container():
+def is_bootc_host():
     """Returns true is the system is managed as an immutable container,
        false otherwise.  If msg is True, a warning message is displayed
        for the user.
     """
+    ostree_booted = '/run/ostree-booted'
+    usr = '/usr/'
```

Comment:
Minor/optional: not sure this is worth making a variable.

Reply:
true, will fix in the next push.

Reply:
I like the variables defined for readability, but either way is fine by me.